script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt

```diff
@@ -57,20 +57,20 @@ The wooden box opens with a soft, dry sound.
 Inside there is a thin iron rod, about 15 centimeters long.
 <Page16>
 "It's... an iron rod."
-A plain iron rod with no signs it has ever been used.
+A plain iron rod that looks like it's seen regular use.//　とくに飾り気のない、使い込まれた跡のある鉄の棒。
 <Page17>
-...My father must have really hated me to leave me a piece of junk like this.
+...With an antique heirloom like this, my father must have really hated me.//　……こんな骨董品が形見とは、親父はよっぽど俺が気に食わなかったとみえる。
```

Comment:
Missed this the first time, but is `heirloom` supposed to be encoding the action of the thing being left to Shiki in Makihisa's will? I feel like that action might need to be represented a bit more strongly, since heirloom just sorta reads like an object to me, it reads like the existence of the object is what means his father hated him rather than the fact that object is what his father chose to leave to him